File: src/main/java/de/maggiwuerze/xdccloader/persistence/ChannelRepository.java
package de.maggiwuerze.xdccloader.persistence;

import de.maggiwuerze.xdccloader.model.entity.Channel;
import java.util.List;
import org.springframework.data.repository.CrudRepository;
import org.springframework.data.repository.PagingAndSortingRepository;
import org.springframework.stereotype.Repository;


@Repository
public interface ChannelRepository extends CrudRepository<Channel, Long> {

	List<Channel> findAll();
}

File: src/main/kotlin/de/maggiwuerze/xdccwebloader/persistence/ChannelRepository.kt
package de.maggiwuerze.xdccwebloader.persistence

import de.maggiwuerze.xdccwebloader.model.entity.Channel
import org.springframework.data.repository.CrudRepository
import java.util.*

@org.springframework.stereotype.Repository
interface ChannelRepository : CrudRepository<Channel, UUID> {
    override fun findAll(): List<Channel>
}